File: reddio-csharp/reddio.api/V1/Rest/GetNonceResponse.cs
using System;
using System.Text.Json.Serialization;

namespace Reddio.Api.V1.Rest
{
    public class GetNonceResponse
    {
        public GetNonceResponse()
        {
        }

        public GetNonceResponse(long nonce)
        {
            Nonce = nonce;
        }

        [JsonPropertyName("nonce")] public Int64 Nonce { get; set; }
    }
}

File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/GetNonceResponse.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class GetNonceResponse {
    @JsonProperty("nonce")
    public long nonce;
}
